HIP/hip-1064.md

```diff
@@ -0,0 +1,165 @@
+---
+hip: 1064
+title: Daily Rewards For Active Nodes
+author: Neeharika Sompalli <@Neeharika-Sompalli>
+working-group: Leemon Baird (@lbaird), Richard Bair <@rbair23>, Michael Tinker <@tinker-michaelj>
+requested-by: Hashgraph
+type: Standards Track
+category: Core
+needs-council-approval: Yes
+status: Draft
+last-call-date-time: 2024-04-08T07:00:00Z
+created: 2024-10-17
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/1063
+updated: 2024-10-17
+---
+
+## Abstract
+
+This HIP proposes a reward mechanism that will incentivize nodes to remain active on the network. The definition of active node is described below. The minimum reward per day for nodes will be a configurable property decided by the council. This minimum reward may be adjusted based on fees already paid throughout the day.
+
+## Motivation
+
+The goal is to give a strong incentive for node operators to keep them active and properly configured throughout every staking period.
+
+## Rationale
+
+Clearly defining the criteria for node activity, we can ensure that rewards are distributed fairly, incentivizing nodes to remain active and contribute to the network’s stability and growth.
+
+### Terminology
+
+In the Hashgraph consensus algorithm, nodes communicate by sharing information through events. An event contains some transactions.
+
+In each round, each node creates a *witness* which is an event created in that round that does not have any descendants from the same node in that round. A *judge* is a specific type of witness event that gains importance because it is known and agreed upon by a majority of the network.
```

Comment:
I did not understand this statement:

> In each round, each node creates a *witness* which is an event created in that round that does not have any descendants from the same node in that round

Maybe it is too much detail in this HIP to define exactly what a witness or a judge is. Perhaps we can just say something like:

"In the hashgraph algorithm, honest nodes that are online and contributing to consensus will regularly create unique famous witnesses, called judges, in nearly every round." That might be sufficient and accurate enough (although we should definitely review the language with Leemon).

Reply:
Updated as per the suggestion. Thanks Richard

Reply:
Confirmed with Leemon and updated the definition as per glossary.